_meta/beat.yml

```diff
@@ -27,6 +27,11 @@ apm-server:
   # Maximum number of new connections to accept simultaneously (0 means unlimited)
   # max_connections: 0
 
+  # Tracing support for the server's HTTP endpoints and event publisher.
+  tracing:
```

Comment:
should be commented out since it's the default

Reply:
fixed